ui/src/app/admin/controllers/AdminCtrl.js

```diff
@@ -46,8 +46,9 @@
     };
 
     $scope.approve = function(username) {
-      AdminService.approve(username).then(function (response) {
-        ToastService.showToast('Approved', username);
+      AdminService.approve(username.user_name).then(function (response) {
```

Comment:
`username.user_name` looks ugly, shall we change the param to `user` and get as `user.user_name`?

Reply:
Done. I'll be a little more careful from the next time with variable and parameter names :sweat_smile: